fastlane_bot/helpers/txhelpers.py

```diff
@@ -179,16 +135,30 @@ def validate_and_submit_transaction(
         else:
             current_gas_price = arb_tx["gasPrice"]
 
-        signed_arb_tx = self.sign_transaction(arb_tx)
+        signed_arb_tx = self.web3.eth.account.sign_transaction(arb_tx, self.ConfigObj.ETH_PRIVATE_KEY_BE_CAREFUL)
```

Comment:
what happened to self.sign_tx? why was it removed?

Reply:
Clear up code.
It was a single-line function used only in two places in the code.
Placing that single line directly in each one of these 2 places, makes it slightly easier to follow the code, and also cleans it up a bit.

Reply:
fwiw I disagree. the original code introduced an abstraction for signing transactions which was a/ deliberate, and b/ IMO usually a good thing. we can discuss this -- but removing abstractions is not often warranted

Reply:
Noted.